package.json

```diff
@@ -57,6 +57,7 @@
     "@emotion/core": "^11.0.0",
     "@emotion/react": "^11.1.4",
     "@emotion/styled": "^11.0.0",
+    "@material-ui/core": "^5.0.0-alpha.29",
```

Comment:
while we are at it, shouldn't we delete bootstrap, react-bootstrap and shards-react?

Reply:
I expect this to be an ongoing process. We have places where we're using react-bootstrap and shards now. If we remove them, we have to put a lot of effort into refactoring all the places where we use it. I think it's better to do this as we go along.

Reply:
Alrighty